### DISCUSSION TITLE
Clean install

### DISCUSSION THREAD

**Discussion by yutaka-takaoka** [HumanSupportRobot/QandA#14] [Q&A] [answered]
I tried to do the clean install with the image ( [hsrb-installer_22.04.01_focal_HSRC-PHASE1_en_US.UTF-8.iso](https://packages.hsr.io/images/hsrb-installer_22.04.01_focal_HSRC-PHASE1_en_US.UTF-8.iso)) and the HSRC manual said that it starts installing automatically, but it didn't. Actually it asked me to input "user" or "computer name". Even after installing it somehow, I couldn't find "hsrb_command" for restoring configuration files.
Any advice on this?

**Comment by hsr-admin** [comment, marked as answer]
There is the possibility that the USB media was selected using UEFI mode. Select the USB media using “Legacy” mode (the mode without UEFI), then perform the install again.

reference)
 https://docs.hsr.io/hsr_faq/hsr_common_faq.html#id1
 https://docs.hsr.io/hsr_faq_en/hsr_common_faq.html#id1

**Reply by yutaka-takaoka** [reply]
That's it. Thanks a lot! I will check the FAQ first.
